source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -0,0 +1,263 @@
+|`  <parameter name='IP' value='10.0.0.3'/>`    | `    {                       ` |
+|                                              | `      'name': 'IP',         ` |
+|                                              | `      'value': '10.0.0.3'   ` |
+|                                              | `    },                      ` |
+|                                              | `  ]                         ` |
+|`</filterref>`                                | `},                          ` |
+
+The following table shows an example of the previously implemented mapping of the reference to a top-level filter of a network interface in libvirt XML to VDSM json:
+
+| libvirt XML fragment | VDSM json fragment|
+|-------------|-----------|
+|`<filterref filter='clean-traffic'/>`| `'filter': 'clean-traffic',`|
+
+Implemented is a combination of the naive mapping and the previously implemented mapping, which gives a backward compatible extension of the VDSM API:
+
+| libvirt XML fragment                         | VDSM json fragment             |
+|----------------------------------------------|--------------------------------|
+| `<filterref filter='clean-traffic'>`         | `  'filter': 'clean-traffic',` |
+|                                              | `  'filterParameters': [     ` |
+|`  <parameter name='IP' value='10.0.0.1'/>`   | `    {                       ` |
+|                                              | `      'name': 'IP',         ` |
+|                                              | `      'value': '10.0.0.1'   ` |
+|                                              | `    },                      ` |
+|`  <parameter name='IP' value='10.0.0.2'/>`   | `    {                       ` |
+|                                              | `      'name': 'IP',         ` |
+|                                              | `      'value': '10.0.0.2'   ` |
+|                                              | `    },                      ` |
+|`  <parameter name='IP' value='10.0.0.3'/>`   | `    {                       ` |
+|                                              | `      'name': 'IP',         ` |
+|                                              | `      'value': '10.0.0.3'   ` |
+|                                              | `    },                      ` |
+|                                              | `  ],                        ` |
+|`</filterref>`                                | `                            ` |
+
+
+The new key `filterParameters` is only be considered by VDSM, if a value for `filter` is set. An older VDSM, which does not know about `filterParameters`, just ignores it.
+
+### Event Reporting
+
+#### Errors
+
+* Try to set invalid parameters:
+  * Invalid name, e.g. the name contains invalid characters
+  * Forbidden name, e.g. the user wants to set the parameter `MAC`, which should be set only by libvirt, but not by the user
+* Try to set invalid values:
+  * Unexpected data type
```

Comment:
not clear to me - we explicitly said that we are not to validate data types. what do we refer to here?

actually, I do not understand the purpose of this whole Errors section.